test/hotspot/gtest/jfr/test_adaptiveSampler.cpp

```diff
@@ -47,6 +45,8 @@
 
 #include "unittest.hpp"
 
+#include <cmath>
```

Comment:
Why is this after unittest.hpp?

Reply:
System includes go after HotSpot includes.